inlong-sort-standalone/sort-standalone-source/src/main/java/org/apache/inlong/sort/standalone/sink/elasticsearch/EsCallbackListener.java

```diff
@@ -0,0 +1,120 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.inlong.sort.standalone.sink.elasticsearch;
+
+import java.util.List;
+
+import org.apache.inlong.sort.standalone.channel.ProfileEvent;
+import org.apache.inlong.sort.standalone.utils.InlongLoggerFactory;
+import org.elasticsearch.action.DocWriteRequest;
+import org.elasticsearch.action.bulk.BulkItemResponse;
+import org.elasticsearch.action.bulk.BulkProcessor;
+import org.elasticsearch.action.bulk.BulkRequest;
+import org.elasticsearch.action.bulk.BulkResponse;
+import org.slf4j.Logger;
+
+/**
+ * EsCallbackListener
+ *
+ */
+public class EsCallbackListener implements BulkProcessor.Listener {
+
+    public static final Logger LOG = InlongLoggerFactory.getLogger(EsCallbackListener.class);
+
+    private EsSinkContext context;
+
+    /**
+     * Constructor
+     * 
+     * @param context
+     */
+    public EsCallbackListener(EsSinkContext context) {
+        this.context = context;
+    }
+
+    /**
+     * beforeBulk
+     * 
+     * @param executionId
+     * @param request
+     */
+    @Override
+    public void beforeBulk(long executionId, BulkRequest request) {
+//      LOG.info("beforeBulk,executionId:" + executionId + ",request:" + request);
+    }
+
+    /**
+     * afterBulk
+     * 
+     * @param executionId
+     * @param request
+     * @param response
+     */
+    @Override
+    public void afterBulk(long executionId, BulkRequest request, BulkResponse response) {
+//      LOG.info("afterBulk,executionId:" + executionId + ",request:" + request + ",response:" + response);
+        BulkItemResponse[] itemResponses = response.getItems();
+        List<DocWriteRequest<?>> requests = request.requests();
+        int itemSize = Math.min(itemResponses.length, requests.size());
+        if (itemResponses.length != requests.size()) {
```

Comment:
item size can be assigned to one of itemResponses.length  or requests.size() in the case that them are the same value. The comparison of Math.min is only need to operate in the unequal case

Reply:
use itemResponses.length